### ISSUE TITLE
About  loss function

### ISSUE THREAD

**Issue by ghost** [fuy34/superpixel_fcn#15]
I noticed that in file loss.py, you say ' # empirically we find timing 0.005 tend to better performance'. 
Do the results in the paper adopt this strategy? Do the file SpixelNet_bsd_ckpt.tar in pretrain_ckpt folder  adopt this strategy?

**Comment by fuy34** [comment]
A short answer is yes. 

We found this while we explored the deep supervision network. More specifically, we asked the U-Net to provides probability prediction at every level (in total 4 levels), and applied the same loss to each level with a weight. We found it improved the performance a little bit, and ```0.005``` is the weight for the last level at that time. Later, for simplicity,  we turned back to the single output but kept this weight, the performance only slightly dropped and being mildly better than the one without this weight. Therefore, we keep this weight in our final version. But all the performance changes are not significant, and all the version provides quite good results.

**Comment by ghost** [comment]
Thank you for your quick reply，the problem has been solved.
